### ISSUE TITLE
Conflict with function-name-case rule

### ISSUE THREAD

**Issue by soullivaneuh** [kristerkari/stylelint-scss#377]
# Problem

Camel case function imported from vendor are lower-cased by this rule:

```scss
@import "bulma/sass/utilities/functions";
$coral: #e14a4b;
$coral-invert: findColorInvert($coral);
```

In this example, the function will be turned to `findcolorinvert`.

The expected behavior would be to not be touched at all, as the function is declared like this.

# Workaround

I first search a way to disable the rule for `.scss` files, but I didn't see anything like this on `stylelint` configuration.

The only possible workaround is to completely disable this rule.

I'm not sure it's an issue about this plugin or stylelint itself, so I first opened this issue to get some opinions.

**Comment by kristerkari** [comment]
Thanks @soullivaneuh!

You should post the same issue to https://github.com/stylelint/stylelint as the rule if part of stylelint itself and there is nothing we can do about it here :)

**Comment by soullivaneuh** [comment]
I was guessing that, thanks for the confirmation. :+1:
